test/system/test_tpm2_createprimary.sh

```diff
@@ -53,6 +55,11 @@ cleanup
 # Keep the algorithm specifiers mixed to test friendly and raw
 # values.
 for gAlg in 0x04 sha256 0x0C; do
```

Comment:
Why not just change the function to give you all supported hash algorithms.... then you don't need this filter approach. You just loop over all supported hash algorithms